.drone.star

```diff
@@ -1911,7 +1916,7 @@ def copyFilesForUpload():
         ],
     }]
 
-def runWebuiAcceptanceTests(suite, alternateSuiteName, filterTags, extraEnvironment, browser, visualTesting, screenShots):
+def runWebuiAcceptanceTests(suite, alternateSuiteName, filterTags, extraEnvironment, browser, visualTesting, screenShots, earlyFail):
```

Comment:
`earlyFail` is not used in this function. Can it be removed?

Reply:
yes.